### PR TITLE
Fix: Updated contributing README for running in development

### PR DESCRIPTION
**Pull request by katrieltsepelevish**
**Updated contributing README by adding the following:**
- Added missing `build` step before seeding
- Added troubleshooting for `PORT already in use` issue

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -92,3 +103,24 @@ branch are tagged into a release monthly.
 - If your PR refers to or fixes an issue, be sure to add `refs #XXX` or `fixes #XXX` to the PR description. Replacing `XXX` with the respective issue number. See more about [Linking a pull request to an issue
   ](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
 - Be sure to fill the PR Template accordingly.
+
+## Troubleshooting
+
+### No PostHog API key, so analytics won't work
```
Comment (matt-aitken) [inline CONTRIBUTING.md]:
```suggestion
### EADDRINUSE: address already in use :::3030
```

The PostHog bit is actually just a console.log, not an error. It's unrelated to the issue you were having.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -92,3 +103,24 @@ branch are tagged into a release monthly.
...
+
+When receiving the following error message:
+``
+webapp:dev: No PostHog API key, so analytics won't work
```
Comment (matt-aitken) [inline CONTRIBUTING.md]: Remove this line, this is unrelated.

**Review by matt-aitken — CHANGES_REQUESTED** [review]
This is great, just a small change to the title of the troubleshooting section.

File: CONTRIBUTING.md (hunk 1 above)
Comment (katrieltsepelevish) [reply CONTRIBUTING.md]: @matt-aitken Fixed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (katrieltsepelevish) [reply CONTRIBUTING.md]: @matt-aitken Fixed.

**Review by matt-aitken — APPROVED** [review]
